presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -898,6 +901,8 @@ class SystemConfig : public ConfigBase {
 
   bool prestoNativeSidecar() const;
   std::string prestoDefaultNamespacePrefix() const;
+
+  std::string poolType() const;
```

Comment:
Another question... This config could be a good fit for node.config instead of system config ? Is there a reason for not considering that option ?